test/integration/coscheduling_test.go

```diff
@@ -229,8 +229,8 @@ func TestCoschedulingPlugin(t *testing.T) {
 					midPriority).Label(coschedulingutil.PodGroupLabel, "pg2-2").ZeroTerminationGracePeriod().Obj(), pause),
 			},
 			podGroups: []*v1alpha1.PodGroup{
-				util.MakePG("pg2-1", ns.Name, 3, nil, nil),
-				util.MakePG("pg2-2", ns.Name, 4, nil, nil),
+				util.MakePG("pg2-1", ns.Name, 3, nil, &v1.ResourceList{v1.ResourceMemory: resource.MustParse("150")}),
+				util.MakePG("pg2-2", ns.Name, 4, nil, &v1.ResourceList{v1.ResourceMemory: resource.MustParse("400")}),
```

Comment:
I'd suggest duplicating this test with minResource rather than replacing the existing one.

Reply:
Got it